jax/_src/xla_bridge.py

```diff
@@ -211,9 +211,13 @@ def _version_check(name, get_version, get_build_version,
       scale_for_comparison=100,
   )
   _version_check("cuFFT", cuda_versions.cufft_get_version,
-                 cuda_versions.cufft_build_version)
+                 cuda_versions.cufft_build_version,
+                 # Ignore patch versions.
+                 scale_for_comparison=100)
   _version_check("cuSOLVER", cuda_versions.cusolver_get_version,
-                 cuda_versions.cusolver_build_version)
+                 cuda_versions.cusolver_build_version,
+                 # Ignore patch versions.
+                 scale_for_comparison=100)
   _version_check("cuPTI", cuda_versions.cupti_get_version,
```

Comment:
Maybe do it for cuPTI too?

Reply:
CUPTI has a different version numbering scheme. Versions are monotonically increasing integers.